Update README1.md

rybakma1 opened the pull request.

rybakma1 committed:
Update README1.md

rybakma1 committed:
Update index.html

rybakma1 committed:
Update index.html

rybakma1 committed:
Merge branch 'main' of https://github.com/rybakma1/vf-rybakov-maksim-2 into main

rybakma1 committed:
moved the project to Gulp

Создал компоненты:
- хедер
- футер
- кнопку
- хлебные крошки
- пагинацию на странице

rybakma1 committed:
rewrote the markup by BEM

Переписал классы в соответствии с методологией БЭМ

rybakma1 committed:
Added animation for: hover and: focus

rybakma1 committed:
Added animation page with puctures

Пока только картинки без анимации

rybakma1 committed:
Added animation to animation page

Мне кажется, нормально получилось.

rybakma1 committed:
Added SVG page with SVG IMG

Добавил SVG картинку

rybakma1 committed:
Added adaptive to the main page

Мобильное меню сделал раскрывающимся за счет js. Только стилями тоже можно, но тогда кнопка на слайдере не кликабельной становится из-за того, что слой мобильного меню выше слоя остального кода.

rybakma1 committed:
Added adaptive to the popup

rybakma1 committed:
Added adaptive to the "blog" page and "contact us" page

rybakma1 committed:
Added adaptive to the "catalog" page

rybakma1 committed:
Added adaptive to the "toys" page

rybakma1 committed:
Added adaptive to the "Animation" page

rybakma1 committed:
Added cropping and retinizing to content images

Только фоновым картинкам не добавлял ни кадрирование, ни ретинизацию

rybakma1 committed:
Added a grid to the category products block of the catalog page

Изначально созданный блок не удалял пока для сравнения

rybakma1 committed:
Added audio to the SVG page

rybakma1 committed:
Added Grid page

Balykov1987 commented on src/components/blog-card/_blog-card.scss:
```diff
@@ -0,0 +1,45 @@
+        line-height: 25px;
+        color: #1E0606;
+    }
+}
```
Добавляй пустую строку чтоб не возникало ошибок.

Balykov1987 commented on src/pages/catalog/style.scss:
```diff
@@ -134,6 +134,187 @@
+}
+
+
+.row {
+    .content {
+    }
+}
```
Не размещай пустые классы без стилей

rybakma1 replied:
Исправил - удалил лишний класс

Balykov1987 commented on src/pages/blog/index.js:
```diff
@@ -1 +1,21 @@
+ 
+buttonMobileHeader.addEventListener('click', function () {
+    mobileHeader.classList.add('mobile-menu__open')
+    buttonMobileHeader.classList.add('displey-none')
+    buttonCloseMobileHeader.classList.remove('displey-none')
```
Старайся писать правильные названия на английском - правильно будет display-none

rybakma1 replied:
Исправил во всех js файлах, в файле header.html и в стилевом файле хедера.

Balykov1987 commented on src/components/breadcrumbs/index.html:
```diff
@@ -0,0 +1,9 @@
+<div>
+    <div>
+        <div>
+            <div>
+
+            </div>
+        </div>
+    </div>
+</div>
```
Для чего пустые блоки?

rybakma1 replied:
Удалил - это было лишнее. Видимо что-то собирался там сделать, но в итоге по другому сделал.

Balykov1987 commented on src/components/header/index.html:
```diff
@@ -49,4 +49,6 @@
             <a class="header__contact_button" href="/pages/contact-us">Contact us</a>
         </div>
     </div>
+    
+
```
Старайся не делать лишних пробелов

rybakma1 replied:
В файле header.html исправил, но вполне возможно, что в других файлах еще такое встретишь. Буду обращать внимание на это.

Balykov1987 commented on src/pages/animation/index.html:
```diff
@@ -19,43 +19,31 @@ <h1 class="page-title container"> Animation</h1>
-            <div class="animation__taxi_group">
-                <img src="/static/images/taxi.png" alt="taxi">
-                <img class="animation__taxi_message" src="/static/images/taxi-message.png" alt="taxi message">
+    <section class="">
```
Не оставляй пустой класс

rybakma1 replied:
В этом месте секшену дал класс animation, а один вложенный ДИВ удалил - он лишним там оказался.

Balykov1987 commented on src/pages/catalog/style.scss:
```diff
@@ -166,15 +160,19 @@
+            font-size: 36px;
+            line-height: 44px;
         }
         @media (min-width: 801px) {
```
Придерживайся стандартных медиа запросов

rybakma1 replied:
Ага, уже в процессе понял, что лучше к стандартным брейкпоинтам привязываться.

rybakma1 committed:
Add SVG animation

Солнце заходит и выходит, при этом фон становится темнее при закате и светлее при восходе солнца.

rybakma1 committed:
Makes corrections for validation

- Добавил пустую последнюю строку в файлах.
- Заменил тег <p> на <time> у карточек на странице blog.
- Добавил ссылки к иконкам социальных сетей в футере.

rybakma1 committed:
Changes checkbox color on the Toys page

rybakma1 committed:
Fixes the adaptive of "about us" block on the main page

Убрал отрицательный маргин, из-за которого на мобильном разрешении появлялся отступ в 20рх справа.

rybakma1 committed:
Fixes the position of the slider pagination buttons

На мобильном разрешении паддинг изменил у баттонов
